Makefile

```diff
@@ -158,7 +158,7 @@ test-unit-cover: ARGS=-timeout=10m -tags='$(UNIT_TEST_TAGS)' -coverprofile=$(TES
 test-unit-cover: TEST_PACKAGES=$(PACKAGES_UNIT)
 test-race: ARGS=-timeout=10m -race -tags='$(TEST_RACE_TAGS)'
 test-race: TEST_PACKAGES=$(PACKAGES_UNIT)
-test-e2e: ARGS=-timeout=25m -v --tags='$(TEST_E2E_TAGS)'
+test-e2e: ARGS=-timeout=25m -v -tags='$(TEST_E2E_TAGS)'
```

Comment:
wow, our e2e tests didn't fully work?

Reply:
We are not add experimental e2e test for now, 

- we need to create experimental docker build `EXPERIMENTAL=true make install (https://github.com/umee-network/umee/issues/1770)